.github/workflows/OCV-Contrib-PR-3.4-U20.yaml

```diff
@@ -1,51 +1,64 @@
-name: PR:3.4 U20
+name: OCV Contrib PR:3.4 U20
 
-on: workflow_call
+on:
+  pull_request:
+    branches:
+      - main
+  workflow_call:
 
 env:
-  EXTRA_CMAKE_OPTIONS: '-DBUILD_DOCS=ON -DPYTHON_DEFAULT_EXECUTABLE=/usr/bin/python3 -DBUILD_EXAMPLES=ON -DOPENCV_ENABLE_NONFREE=ON -DENABLE_CCACHE=OFF'
-  OPENCV_TEST_DATA_PATH: '/opencv_extra/testdata'
-  OPENCV_CONTRIB_DOCKER_WORKDIR: '/__w/opencv_contrib/opencv_contrib'
+  EXTRA_CMAKE_OPTIONS: '-DBUILD_DOCS=ON -DPYTHON_DEFAULT_EXECUTABLE=/usr/bin/python3 -DOPENCV_DOWNLOAD_PATH=/home/ci/binaries_cache -DBUILD_EXAMPLES=ON -DOPENCV_ENABLE_NONFREE=ON'
   PR_AUTHOR: ${{ github.event.pull_request.user.login }}
   PR_AUTHOR_FORK: ${{ github.event.pull_request.head.repo.full_name }}
   SOURCE_BRANCH_NAME: ${{ github.head_ref }}
   TARGET_BRANCH_NAME: ${{ github.base_ref }}
   ANT_HOME: '/usr/share/ant'
-  PYTHONPATH: /opencv-contrib-build/python_loader:$PYTHONPATH
+  GIT_CACHE_DOCKER: '/home/ci/git_cache'
+  PYTHONPATH: /build/python_loader:$PYTHONPATH
+  OPENCV_TEST_DATA_PATH: '/opencv_extra/testdata'
+  OPENCV_CONTRIB_DOCKER_WORKDIR: '/opencv_contrib'
+  DNN_MODELS: '/home/ci/dnn-models'
   GTEST_FILTER_STRING: '-tracking_GOTURN.GOTURN/*'
 
 jobs:
   BuildAndTest:
-    runs-on: ubuntu-20.04
+    runs-on: opencv-cn-lin-x86-64
     defaults:
       run:
         shell: bash
     container:
-      image: quay.io/asenyaev/opencv-ubuntu:20.04
+      image: quay.io/opencv-ci/opencv-ubuntu:20.04
+      volumes:
+        - /home/opencv-cn/git_cache:/home/ci/git_cache
+        - /home/opencv-cn/ci_cache/opencv_contrib:/home/ci/.ccache
+        - /home/opencv-cn/binaries_cache:/home/ci/binaries_cache
+        - /home/opencv-cn/dnn-models:/home/ci/dnn-models
     steps:
+    - name: Setup infra environment
+      if: ${{ github.event.repository.name != 'opencv-gha-workflow' }}
+      run: if [[ ${{ github.event.repository.name }} == 'ci-gha-workflow' ]]; then echo "TARGET_BRANCH_NAME=3.4" >> $GITHUB_ENV; fi
```

Comment:
These lines have one variable `${{ github.event.repository.name }}`, but have the different output in case of renaming of this repository from `opencv-gha-workflow` to `ci-gha-workflow`. Created a ticket to GitHub Support.